surround/remote/cli.py

```diff
@@ -240,25 +228,6 @@ def parse_remote_args(remote_parser, parsed_args):
             else:
                 print("error: not a surround project")
 
-def parse_add_args(parsed_args):
-    """
-    Executes the "add" sub-command which adds a file to a specified remote.
-
-    :param parsed_args: arguments parsed from the user (will contain the file path and remote name)
-    :type parsed_args: <class 'argparse.Namespace'>
-    """
-
-    if is_surround_project():
-        actual_current_dir = os.getcwd()
-        os.chdir(get_project_root_from_current_dir())
```

Comment:
We need this if we are adding from subdir

Reply:
Happens during push and pull commands, so the add method will work when called in there